BSPConvert.Lib/Source/EntityConverter.cs

```diff
@@ -209,6 +213,80 @@ private void ConvertFuncStatic(Entity funcStatic)
 			funcStatic.ClassName = "func_brush";
 		}
 
+		private void ConvertFuncPlat(Entity entity)
+		{
+			var moveDistance = 0f;
+			var brushThickness = GetBrushThickness(entity);
+
+			if (float.TryParse(entity["height"], out var height))
+				moveDistance = height + brushThickness;
+			else if (float.TryParse(entity["lip"], out var lip))
+				moveDistance = -(lip - 2 - (brushThickness * 2));
```

Comment:
What does the `- 2` mean? You should declare a constant with a name that describes what it is being used for.